### PR TITLE
Refine sshclient and test_nvme

### PR DESCRIPTION
**Pull request by chenge3**
1. Refine sshclient to print less trying to connect info
2. Refine test_nvme to make ssh as global variable

### CODE REVIEW

**Comment by emcinfrasim** [comment]
<details><summary>Test Error Details</summary><p>
TestName: test_set_drive_page_file_exist 
<br>ErrorType: exceptions.AssertionError 
<br>ErrorMessage: line None, 
-------------------- >> begin captured logging << --------------------
0Model: INFO: Fill OEM json file
0Model: INFO: oem_data.json not exist or nvme_map_dict is empty
--------------------- >> end captured logging << --------------------- 
<br> 
<br></p></details>

File: test/functional/test_nvme.py [hunk 1]
```diff
@@ -65,26 +69,20 @@ def tearDownClass(cls):
 
     def get_nvme_disks(self):
         # Return nvme , eg. ['/dev/nvme0n1', '/dev/nvme0n2']
-        ssh = sshclient.SSH("127.0.0.1", "root", "root", port=2222)
```
Comment (weia5-emc) [inline test/functional/test_nvme.py]: clear code. 👍

File: infrasim/sshclient.py [hunk 2]
```diff
@@ -34,24 +34,28 @@ def __del__(self):
...
+                            host=self.host_ip,
+                            port=self.host_port))
         start = time.time()
         while True:
             if self.connect():
                 break
             end = time.time()
             if end - start > timeout:
                 return False
+        print("Connected to SSH server: {username}@{host}:{port}"
```
Comment (weia5-emc) [inline infrasim/sshclient.py]: 🆒 , less message on console.

File: test/functional/test_nvme.py [hunk 3]
```diff
@@ -40,9 +40,13 @@ def start_node():
         node.init()
         node.precheck()
         node.start()
-        time.sleep(10)
+        node.wait_node_up()
         helper.port_forward(node)
 
+        ssh = sshclient.SSH("127.0.0.1", "root", "root", port=2222)
+        assert ssh.wait_for_host_up() is True
+        global ssh
```
Comment (weia5-emc) [inline test/functional/test_nvme.py]: declare global variable before use it.

**Comment by chenge3** [comment]
@weia5-emc good catch, code updated.

**Pull request merged**